.editorconfig

```diff
@@ -0,0 +1,158 @@
+# EditorConfig is awesome:http://EditorConfig.org
+# .NET coding convention settings for EditorConfig
+# https://docs.microsoft.com/en-us/visualstudio/ide/editorconfig-code-style-settings-reference
+#
+# The .editorconfig comes from Roslyn and CoreFX repositories:
+# https://github.com/dotnet/corefx/blob/master/.editorconfig
+# https://github.com/dotnet/roslyn/blob/master/.editorconfig
+
+# Top-most EditorConfig file
+root = true
+
+# Don't use tabs for indentation.
+[*]
+indent_style = space
+# (Please don't specify an indent_size here; that has too many unintended consequences.)
+
+# Code files
+[*.{cs,csx,vb,vbx}]
+indent_size = 4
+insert_final_newline = true
+charset = utf-8
+
+# Xml project files
+[*.{csproj,vbproj,vcxproj,vcxproj.filters,proj,projitems,shproj}]
```

Comment:
what about ps1xml files?  Should we remove extensions we don't use in this repo?

Reply:
Added resx,ps1xml. Removed unused.